examples/adaptiveprecision-blockjacobi/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
-add_executable(adaptiveprecision-blockjacobi adaptiveprecision-blockjacobi.cpp)
-target_link_libraries(adaptiveprecision-blockjacobi ginkgo)
-target_include_directories(adaptiveprecision-blockjacobi PRIVATE ${PROJECT_SOURCE_DIR})
+set(target_name "adaptiveprecision-blockjacobi")
+add_executable(${target_name} ${target_name}.cpp)
+target_link_libraries(${target_name} Ginkgo::ginkgo)
+target_include_directories(${target_name} PRIVATE ${PROJECT_SOURCE_DIR})
```

Comment:
Do we even need the target_name somewhere? Otherwise, we could remove these changes entirely.

Reply:
No, just locally in each example. But I would prefer to keep these changes instead of repeating the different names for the different examples.

Reply:
Except for the ginkgo -> Ginkgo::ginkgo change, this makes no functional changes and I feel like it adds a bit of additional complexity to the CMakeLists.txt. Why not checkout the previous state and just update the Ginkgo target name? Again, these are the files that users are most likely to copy-paste into their CMake code, so keeping them simple and un-surprising feels important to me.

Reply:
okay. I will change everything back to as it was before